src/js/tabs/receiver.js

```diff
@@ -542,7 +542,7 @@ TABS.receiver.initialize = function (callback) {
             }
 
             if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
-                const elrsUidChars = $('span.elrsUid')[0].innerText.split(',').map((uidChar) => {
+                const elrsUidChars = $('span.elrsUid')[0].innerText.split(',').map(uidChar => parseInt(uidChar, 10);
                     return parseInt(uidChar, 10);
```

Comment:
please also remove two next lines. They didn't show up in my review.
Please also squash commits otherwise we can't merge.

Reply:
Done.